src/Controller/Admin/BlogController.php

```diff
@@ -30,6 +30,7 @@
  * purposes. However, in your real Symfony application you should use any of the
  * existing bundles that let you generate ready-to-use backends without effort.
  *
+ * See https://symfony.com/bundles/EasyAdminBundle/current/index.html
  * See http://knpbundles.com/keyword/admin
```

Comment:
I suggest removing that knpbundles.com link as knpbundles.com does not seem functional anymore (it has 0 bundles referenced in it)